README.md

```diff
@@ -119,8 +161,71 @@ public class ExampleAuthenticationStateHandler extends AuthenticationStateHandle
 
 As noted in the above example, a user is ONLY considered authenticated if `AuthenticationResponse.getSessionToken()` is not null. This `sessionToken` can be exchanged via the [Okta Sessions API](https://developer.okta.com/docs/api/resources/authn.html#session-token) to start an SSO session, but that is beyond the scope of this library.
 
-**NOTE:** `UNKNOWN` is not an actual state in Okta's Authentication state model. The method `handleUnknown` is called when an unimplemented state is reached (this could happen if someone turned on MFA support for your Okta organization but was not previously implemented in your state handler). It also possible Okta has added a new state to the state model, and someone from your organization enabled this new state.  
-
-## Contribute to the Project
-
-Take a look at the (contribution guide)[CONTRIBUTING.md] and the [build instructions wiki](https://github.com/okta/okta-auth-java/wiki/Build-It) (though just cloning the repo and running `mvn install` should get you going).
+**NOTE:** `UNKNOWN` is not an actual state in Okta's Authentication state model. The method `handleUnknown` is called when an unimplemented state is reached (this could happen if someone turned on MFA support for your Okta organization but was not previously implemented in your state handler). It also possible Okta has added a new state to the state model, and someone from your organization enabled this new state.
```

Comment:
suggestion:

`UNKNOWN` is not an actual state in Okta's state model. The method `handleUnknown` is called when an unimplemented or unrecognized state is reached. This could happen if:

* Your handler doesn't have an implementation for the state that was just returned
* Your Okta organization configuration changed, and a new state is now possible (for example, an admin turned on multi-factor authentication)
* Okta added something new to the state model entirely